src/core/services/request_sevice.py

```diff
@@ -23,19 +23,40 @@ async def get_request(
         return await self.request_repository.get(request_id)
 
     async def status_update(self, request_id: UUID, new_status_data: RequestStatusUpdateRequest) -> Request:
-        """Обновить статус заявки."""
-        request = await self.get_request(request_id)
-        if request.status == new_status_data.status:
-            raise HTTPException(status_code=HTTPStatus.BAD_REQUEST, detail=REVIEWED_REQUEST)
-        request.status = new_status_data.status
-        await self.send_message(request.user, request.status)
-        return await self.request_repository.update(id=request.id, request=request)
-
-    async def send_message(self, user: User, status: str) -> None:
-        """Отправить сообщение о решении по заявке в telegram."""
-        if status is Status.APPROVED:
-            return await send_approval_callback(user)
-        return await send_rejection_callback(user)
+        """Обновление статус заявки."""
+        request = await self.check_request_status(request_id)
+        request.status = new_status_data
+        return await self.request_repository.update(request.id, request)
+
+    async def check_request_status(self, id: UUID) -> None:
+        """Уточнение статуса заявки."""
+        request = await self.get_request(id)
+        if request.status in (Request.Status.APPROVED, Request.Status.DECLINED):
+            raise HTTPException(status_code=HTTPStatus.NOT_FOUND, detail=REVIEWED_REQUEST.format(request.status))
+
+        return request
+
+    async def accepted_request_update_status(self, id: UUID) -> Request:
+        """Заявка на участие одобрена.
+
+        - Уточнение текущего статуса заявки.
+        - Обновление статуса заявки.
+        - Уведомление участника об одобрении заявки.
+        """
+        request = await self.status_update(id, Request.Status.APPROVED)
+        await bot_services().notify_approved_request(request.user)
+        return request
+
+    async def declined_request_update_status(self, id: UUID) -> Request:

```

Comment:
```suggestion
    async def declined_request(self, id: UUID) -> Request:
```
В других местах тоже поправь, а то какой-то набор слов